testing/TestHarness/TestHarness.Shared/GlobalUsings.cs

```diff
@@ -47,21 +35,22 @@
 	global using Microsoft.UI.Xaml.Markup;
 	global using Microsoft.UI.Xaml.Data;
 	global using Microsoft.UI.Xaml.Media;
-global using Window = Microsoft.UI.Xaml.Window;
-global using LaunchActivatedEventArgs = Microsoft.UI.Xaml.LaunchActivatedEventArgs;
+	global using Window = Microsoft.UI.Xaml.Window;
+	global using LaunchActivatedEventArgs = Microsoft.UI.Xaml.LaunchActivatedEventArgs;
 #else
-global using Windows.System;
-global using Windows.UI.Xaml;
-global using Windows.UI.Xaml.Controls;
-global using Windows.UI.Xaml.Controls.Primitives;
-global using Windows.UI.Xaml.Input;
-global using Windows.UI.Xaml.Navigation;
-global using Windows.UI.Xaml.Markup;
-global using Windows.UI.Xaml.Data;
-global using Windows.UI.Xaml.Media;
-global using Window = Windows.UI.Xaml.Window;
-global using NavigationView = Microsoft.UI.Xaml.Controls.NavigationView;
-global using NavigationViewSelectionChangedEventArgs = Microsoft.UI.Xaml.Controls.NavigationViewSelectionChangedEventArgs;
+	global using Windows.System;
+	global using Windows.UI.Xaml;
+	global using Windows.UI.Xaml.Controls;
+	global using Windows.UI.Xaml.Controls.Primitives;
+	global using Windows.UI.Xaml.Input;
+	global using Windows.UI.Xaml.Navigation;
+	global using Windows.UI.Xaml.Markup;
+	global using Windows.UI.Xaml.Data;
+	global using Windows.UI.Xaml.Media;
+	global using Window = Windows.UI.Xaml.Window;
+	global using NavigationView = Microsoft.UI.Xaml.Controls.NavigationView;
+	global using NavigationViewSelectionChangedEventArgs = Microsoft.UI.Xaml.Controls.NavigationViewSelectionChangedEventArgs;
+	global using LaunchActivatedEventArgs = Windows.ApplicationModel.Activation.LaunchActivatedEventArgs;
```

Comment:
this is overzealous; it is probably best to keep that using for app.xanml.cs

Reply:
Window and LaunchActivatedEventArgs moved to app.xaml.cs. The other two are used in multiple places so have been left here